resources.go

```diff
@@ -106,6 +106,9 @@ func (a *App) createResources() error {
 		Add(fs.NewResource("delete", contentService.Delete, &fs.Meta{
 			Delete: "/:id",
 			Args:   fs.Args{"id": createArg(fs.TypeUint64, "The content ID")},
+		})).
+		Add(fs.NewResource("bulk-delete", contentService.BulkDelete, &fs.Meta{
+			Post: "/bulk-delete",
 		}))
```

Comment:
We used the `DELETE /:id` for single record deletion. So I think we should use the `DELETE /delete` for the bulk deletion.
```suggestion
		Add(fs.NewResource("bulk-delete", contentService.BulkDelete, &fs.Meta{
			Delete: "/delete",
		}))
```